File: 24/Innovazione.py
from Progetti import Progetti

class Innovazione(Progetti):
    def __init__(self, cod, tipo, titolo, nomeCognomeCoordinatore, organizzazione, importoTotaleInMilioniDiEuro,numeroAziendeCoinvolte) -> None:
        super().__init__(cod, tipo, titolo, nomeCognomeCoordinatore, organizzazione, importoTotaleInMilioniDiEuro)
        self.numeroAziendeCoinvolte = numeroAziendeCoinvolte


    #print("tipo, titolo, codice, coordinatore, organizzazione, argomento,partner, aziende, importo totale in migliaia di euro")  
    def __str__(self) -> str:
        return self.tipo+"\t"+self.titolo+"\t"+str(self.cod)+"\t"+self.nomeCognomeCoordinatore+"\t"+self.organizzazione+"\t"+"-"+"\t"+"-"+"\t"+str(self.numeroAziendeCoinvolte)+"\t"+str(self.importoTotaleInMilioniDiEuro)

File: simu11/src/Innovazione.java
public class Innovazione extends Progetto {
    int num_azziende_coinvolte;

    public Innovazione(int codice, String tipo, String titolo_progetto, String nomeCognome_coordinatore,
            String orgaString, double importo_totale_in_milioni_di_euro, int num_azziende_coinvolte) {
        super(codice, tipo, titolo_progetto, nomeCognome_coordinatore, orgaString, importo_totale_in_milioni_di_euro);
        this.num_azziende_coinvolte = num_azziende_coinvolte;
    }
    @Override
    public String toString() {
        return tipo+" "+titolo_progetto+" "+codice+" "+nomeCognome_coordinatore+" "+orgaString+" - "+" - "+num_azziende_coinvolte+" "+(importo_totale_in_milioni_di_euro *1000);
    }
}
